### DISCUSSION TITLE
Need to add shortcode functionality to pull global colors from custom fields in the global settings of elementor.

### DISCUSSION THREAD

**Discussion by ppandey996** [elementor/elementor#25100] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I want to add the global colors to elementor and that is going to be pulled from the backend custom fields. To achieve this functionality, I need to create shortcode to fetch colors from custom fields and then map it to the elementor global settings but to do that also elementor plugin needs to support shortcode functionality in the global settings section but it's not supporting currently.

### Describe the solution you'd like.

I need the shortcode option to be added in the global settings same as any dynamic field which are present for other fields of elementor. Please check [https://i.imgur.com/l9y18oW.png](https://i.imgur.com/l9y18oW.png)

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by hitsgee** [comment]
I would also love this native with Elementor. For now I use Dynamic Shortcodes to achieve this and more. https://www.dynamic.ooo/dynamic-shortcodes
